libs/@guardian/source-react-components-development-kitchen/src/file-input/FileInput.stories.tsx

```diff
@@ -63,12 +63,7 @@ export const FileTypeValidation: StoryFn<typeof FileInput> = () => {
 
 export const FileSizeValidation: StoryFn<typeof FileInput> = () => {
 	return (
-		<FileInput
-			id="file-input"
-			label="Upload a file"
-			supporting="Only supports very very small images"
-			maxFileSize={1}
-		/>
+		<FileInput id="file-input" label="Upload a file" maxFileSize={5_000_000} />
```

Comment:
Why override the existing story and not create a distinct one?

Reply:
The two stories wouldn't be distinct, they'd be the same here as both variants occur from a max file size being provided so an additional test would be a duplicate. I've removed the supporting text as it is redundant with the new warning message.